src/Microsoft.AspNetCore.Razor.Evolution/CodeGeneration/RedirectedBasicWriter.cs

```diff
@@ -70,7 +72,39 @@ public override void WriteHtmlAttribute(CSharpRenderingContext context, HtmlAttr
 
         public override void WriteHtmlContent(CSharpRenderingContext context, HtmlContentIRNode node)
         {
-            _previous.WriteHtmlContent(context, node);
+            if (context.Options.DesignTimeMode)
```

Comment:
-_- Would it be awful to have a DesignTime redirected writer and a Runtime redirected writer? That would then leave it up to _previous to use the appropriate one.